integration_test/ops_agent_test.go

```diff
@@ -1611,6 +1611,41 @@ func TestWindowsEventLog(t *testing.T) {
 	})
 }
 
+func TestWindowsEventLogWithNonDefaultTimeZone(t *testing.T) {
+	t.Parallel()
+	gce.RunForEachPlatform(t, func(t *testing.T, platform string) {
+		t.Parallel()
+		if !gce.IsWindows(platform) {
+			t.SkipNow()
+		}
+		ctx, logger, vm := agents.CommonSetup(t, platform)
+		if _, err := gce.RunRemotely(ctx, logger.ToMainLog(), vm, "", `Set-TimeZone -Id "Eastern Standard Time"`); err != nil {
+			t.Fatal(err)
```

Comment:
What if the new windows VM default time zone is just the same as Eastern Time.  Does the VM created with zone us-east1 default to have Eastern Time?

Reply:
If the default time zone is already Eastern Time then this command is a no-op and the test should still fail as expected.

In my testing the VM time zone was always UTC regardless of which Cloud zone it was created in.